libsolidity/analysis/TypeChecker.h

```diff
@@ -73,7 +73,12 @@ class TypeChecker: private ASTConstVisitor
 	void checkFunctionOverride(FunctionDefinition const& function, FunctionDefinition const& super);
 	void overrideError(FunctionDefinition const& function, FunctionDefinition const& super, std::string message);
 	void checkContractAbstractFunctions(ContractDefinition const& _contract);
-	void checkContractAbstractConstructors(ContractDefinition const& _contract);
+	void checkContractBaseConstructorArguments(ContractDefinition const& _contract);
+	void annotateBaseConstructorArguments(
```

Comment:
We try to keep the same order of functions in the header and in the cpp file.

Reply:
Done.